examples/src/bin/simple_0rtt_server.rs

```diff
@@ -0,0 +1,107 @@
+//! This is an example server that streams 0-RTT early data from the client.
```

Comment:
I think [`examples/README.md`](https://github.com/rustls/rustls/blob/main/examples/README.md) needs an update for this new example. 

Similarly, the [daily-tests.yml](https://github.com/rustls/rustls/blob/8c04dba680d19d203a7eda1951ad596f5fc2ae59/.github/workflows/daily-tests.yml#L59-L60) has a section that could use an update for this binary.

Reply:
this example doesn't terminate, it keeps accepting connections

Reply:
Good point, and it doesn't have a `--help` or similar. Fair to skip the CI update then :+1: